.github/workflows/cross-version-tests.yml

```diff
@@ -60,8 +60,26 @@ jobs:
         run: |
           conda install python=3.6
           python --version
+      - name: Get cache key
+        env:
+          INSTALL_COMMAND: ${{ matrix.install }}
```

Comment:
I first tried to put `${{ matrix.install }}` in the run step like below, but it didn't work out as I expected because it contains lots of special characters (e.g. quotes, linebreaks, etc.)

```
run: |
    install_command="${{ matrix.install }}"
```